src/extension.js

```diff
@@ -270,7 +270,31 @@ function searchTags()
 
 	//Case 2. Many tags found
 	} else if (displayFiles.length > 0) {
-		vscode.window.showQuickPick(displayFiles, {matchOnDescription: true, matchOnDetail: true})
+
+		// extract available extensions from ctagssh settings if any
+		let conf = vscode.workspace.getConfiguration('ctagssh');
+		let CTagSSH_showExtensions = [];
```

Comment:
**CTagSSH_** prefix used only for global variables

Reply:
done